### ISSUE TITLE
can i host repl.it on my server ?

### ISSUE THREAD

**Issue by jagdishjadeja** [replit/polygott#52]

**Comment by isuckatprogram** [comment]
I don't think so. This is just the docker image for the languages not the evaluation server or the website. You could create your own version of repl.it and use the docker image for the languages.
